contracts/README.md

```diff
@@ -28,15 +28,38 @@ yarn install
 ## Run Tests
 
 ```sh
-npm run tests
+forge test
+```
+
+For increased levels of test output verbosity, increase the number of `v`s. For example:
+```sh
+forge test -v
+```
+or
+```sh
+forge test -vv
+```
+or
+```sh
+forge test -vvv
+```
+or
+```sh
+forge test -vvvv
 ```
 
 ## Run Lints
 
+This is not really required, as formatting happens everytime you commit to the repository.
+
 ```sh
-npm run lint:fix
+forge fmt .
 ```
```

Comment:
nit: i'd just get rid of this section since it's automated, and we can introduce a separate readme for contributions

Reply:
Sure. Removed. We can later discuss what needs to be included in the `Contributions` section.